com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -776,15 +774,16 @@ internal string ScenePathFromHash(uint sceneHash)
         /// </summary>
         internal uint SceneHashFromNameOrPath(string sceneNameOrPath)
         {
-            var addressableKey = sceneNameOrPath;
-
 #if UNITY_EDITOR
-            // NOTE: Only works with addressables fast play mode script
-            addressableKey = UnityEditor.AssetDatabase.AssetPathToGUID(sceneNameOrPath);
-#endif
+            var addressableKey = UnityEditor.AssetDatabase.AssetPathToGUID(sceneNameOrPath);
+
             if (AddressableKeyToHash.TryGetValue(addressableKey, out var externalSceneHash))
             {
-                Debug.Log($"SceneHashFromNameOrPath: {addressableKey} = {externalSceneHash}");
+                return externalSceneHash;
+            }
+#endif
+            if (AddressableKeyToHash.TryGetValue(sceneNameOrPath, out externalSceneHash))
```

Comment:
revert this please, there's more work you'd have to do if you want to support looking up by scene name. We don't store the scene name in this hashmap.

Reply:
and I don't want more regressions in this area

Reply:
I'll revert then, but would prefer that we support both.